README.md

```diff
@@ -4,57 +4,59 @@ ea_restaurant_data_loader is a microservice as part of [ea_restaurant](https://g
 
-## Project Setup
+## Ea_Restaurant_Data Loader Setup:
 
-In Order to build a Phoenix application, we will need a few dependencies installed in our Operating System:
+In order to setup and compile or run test in ea_restaurant_data_loader project you need to follow the following steps:
 
-- The [Erlang/OTP](https://www.erlang.org/downloads) and the [Elixir](https://elixir-lang.org/install.html#windows) programming language
-- a database - Phoenix recommends PostgreSQL , but you can pick others or not use a database at all
-- and other optional packages
+* You need to install the [Erlang/OTP](https://www.erlang.org/downloads) and the [Elixir](https://elixir-lang.org/install.html#windows) programming language
 
-In order to get a Phoenix app running and to install any extra dependencies we might need to install Hex.
+* It is necessary to create the files `dev.secrets.exs`, `test.secret.exs`, `prod.secret.exs` inside the config folder to storage all the information that you
+may not share in version control or any place out of workplace. Inside config folder you going to have a `example_secret.exs` file.
 
-Here’s the command to install Hex:
+In order to use secrets inside secret file you can access them using the following code line:
+    * `Application.get_env(:ea_restaurant_data_loader, :secret_key)` # Will return the secret key from .secret.exs file
 
-`mix local.hex`
 
-Once we have Elixir and Erlang, we are ready to install Phoenix application generator:
+### Dependencies
 
-`mix archive.install hex phx_new`
-
-After that the `phx.new` generator is now available to generate new applications. In order to config a database Phoenix applications use another elixir package, called Ecto. Ecto is a toolkit for data mapping and language integrated query for Elixir.
-
-Now we are ready to create our Phoenix application running the following command:
-
-`mix phx.new ea_restaurant_data_loader`
-
-In order to have the phoenix server ready it’s good to fetch and install dependencies running the command:
-
-* `mix deps.get`
+* In order to get the neccesary dependencies for the project you have to run the command:
+    * mix deps.get
 
 Note: It is important before to install the dependencies to install a C compiler to be able to compile `bcrypt_elixir` dependency, follow the following
 steps:
 * Install a recent version of [Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/) (On the download page, click on tools for Visual Studio and then you will see the link for the build tools for visual studio 2022)
```

Comment:
You can inspect the link in the button to get the url that actually download the correct file.

Reply:
This part was removed.